awx/main/tasks/system.py

```diff
@@ -78,6 +81,9 @@
 
 def dispatch_startup():
     startup_logger = logging.getLogger('awx.main.tasks')
+
+    convert_jsonfields_to_jsonb()
```

Comment:
Is this going to be okay to run on clusters? This will be running multiple versions of the task all at the same time right? Is that okay?

Reply:
I _think_ this is fine.  If you look at the `migrate_json_fields` task, which is ultimately what is called, I set an advisory lock in the database and short-circuit out if the lock is already taken.

Reply:
oh right :100: